Patches/Core/PawnKindDefs_Humanlike/PawnKinds.xml

```diff
@@ -807,4 +955,19 @@
      </value>
   </Operation>
 
+  <Operation Class="PatchOperationConditional">
+   <xpath>/Defs/PawnKindDef[defName="StrangerInBlack"]/skills</xpath>
+   <nomatch Class="PatchOperationAdd">
+     <xpath>/Defs/PawnKindDef[defName="StrangerInBlack"]</xpath>
+     <value>
+       <skills>
+         <li>
+           <skill>Shooting</skill>
+           <range>8~14</range>
```

Comment:
Way too high. Don't care if they're supposed to be "saving" the colony, that doesn't prevent them being a fuck-up.

Reply:
Done.